ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -452,12 +453,14 @@ private ClassLoadContext createVariablesExecutionContext(
      * @param qualifiersAndMetadata Variable snippet qualifiers.
      * @param definedVariables      Variables that were defined.
      * @param globalVarSymbols      All global variable symbols.
+     * @param type                  type of the variables.
      * @return Exported found variable information (name and type)
      */
     private Map<QuotedIdentifier, GlobalVariable> createGlobalVariables(
             String qualifiersAndMetadata,
             Set<QuotedIdentifier> definedVariables,
-            Collection<GlobalVariableSymbol> globalVarSymbols) {
+            Collection<GlobalVariableSymbol> globalVarSymbols,
+            String type) {
```

Comment:
May not matter once the above comment is addressed. But `type` would be a misleading name to give this. Since this is just really the syntax node kind.

Reply:
Updated in 3fc03c0dee2d7091929456e76b603ab332cb786e